tests/bundle/Functional/JsonSchema/Language.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
can you elaborate on this a bit? Why do you need it? New convention?

Reply:
We are introducing JSON schema to most of our APIs in packages that use the newer, kernel-based web request testing. Granted, we could drop this declaration as it is technically not strictly necessary, but on the other hand it also describes our expectations towards shape of API.